lightkube/codecs.py

```diff
@@ -3,8 +3,10 @@
 
 import yaml
 
-from .generic_resource import get_generic_resource, GenericGlobalResource, GenericNamespacedResource
+from .generic_resource import get_generic_resource, GenericGlobalResource, GenericNamespacedResource, create_resources_from_crd
 from .core.exceptions import LoadResourceError
+from .resources.apiextensions_v1 import CustomResourceDefinition
```

Comment:
Few things on this one: I'm trying to remove dependency of lightkube from lightkube models for documentation geneneration. Basically lightkube imports models it needs from `lightkube.core.internal_model` and in that module there is some logic to just mock non existing dependencies. You can do the same here.

Reply:
I think I understood you, but please take a look.

Also, does [this comment](https://github.com/gtsystem/lightkube/pull/34#discussion_r894689853) apply here too?  Do we need some "try import v1; except import v1beta1" logic to make sure we handle either?

Reply:
Yes, we need to assume users are still using k8s 1.15. I'll resolve this comment and add another one in the right place.